metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_config.py

```diff
@@ -214,6 +217,13 @@ def have_table_data_read_permission(self) -> bool:
             "Overrides `project_id_pattern`."
         ),
     )
+    project_labels: List[str] = Field(
+        default_factory=list,
+        description=(
+            "Ingests projects with the specified labels. Use this property if you want to specify what "
+            "projects to ingest based on project-level labels."
```

Comment:
please describe how this option interacts with project_id_pattern and other similar config options